### ISSUE TITLE
<Link> not working on localhost after HighLightInit is added in NextJS

### ISSUE THREAD

**Issue by jmn** [highlight/highlight#5842]
**Describe the bug**
When using NextJS, `<Link>` stops working on the local dev environment `http://localhost:3000` after the Highlight SDK has been initialized.

**To Reproduce**

1. Install a project with NextJS `npx create-next-app@latest`
2. `pnpm add @highlight-run/next highlight.run @highlight-run/react`
3. Initialize the client SDK. 
```ts

// src/app/layout.tsx
import { HighlightInit } from '@highlight-run/next/highlight-init'

export default function RootLayout({ children }: { children: React.ReactNode }) {
	return (
		<>
			<HighlightInit
				projectId={'<PROJECT_ID>'}
				tracingOrigins
				networkRecording={{
					enabled: true,
					recordHeadersAndBody: true,
					urlBlocklist: [],
				}}
			/>

			<html lang="en">
				<body>{children}</body>
			</html>
		</>
	)
}
```
**Expected behavior**
`<Link>` should continue to work.

**Additional context**
- I used NextJS Appdir
- Tested on a Windows machine.
- See https://github.com/jmn/next-highlight for a reproduction of the problem

**Comment by deltaepsilon** [comment]
I've tried to reproduce here:

https://github.com/highlight/highlight/pull/5851/files

I couldn't reproduce. See successful `next/link` use:
https://github.com/highlight/highlight/assets/878947/6f3ef54b-260c-4a36-b3da-dfa8c7ae4573

I tried to reproduce with the sample project, but it seems to be working???

https://github.com/highlight/highlight/assets/878947/ed0a666c-84ef-436c-b982-1b65dca4e56f

@jmn What are you seeing exactly? What does the failure state look like?

**Comment by jmn** [comment]
Thanks for looking into it.

Here's a screen recording of trying to click the link in the sample project where nothing happens.
![msedge_BqO6T4SPGj](https://github.com/highlight/highlight/assets/35616/57417a71-cc39-4b5e-b708-b9e434751087)

Nothing is logged in the web developer tools Console.

Node version: 18.15.0
OS: Windows 11
Browser Firefox 114.0.2, Edge 114.0.1823.67 

Any other information I can provide? Happy to do so.

**Comment by Vadman97** [comment]
Hi @jmn , this may be fixed with #6001
We've published new packages with the fix.
https://www.npmjs.com/package/@highlight-run/next/v/3.1.3
https://www.npmjs.com/package/highlight.run/v/7.3.8
mind giving it a try?
